apps/documents/assets/ChapterNav.jsx

```diff
@@ -4,22 +4,27 @@ var ChapterNavItem = require('./ChapterNavItem')
 var FlipMove = require('react-flip-move')
 
 const ChapterNav = (props) => {
+  const activeKey = props.activeChapter.id || props.activeChapter.key
   return (
     <nav aria-label={django.gettext('Chapter navigation')}>
       <FlipMove easing="cubic-bezier(0.25, 0.5, 0.75, 1)" typeName="ol" className="u-list-reset">
         {
-          props.chapters.map((chapter, index, arr) =>
-            <li key={chapter.id || chapter.key}>
-              <ChapterNavItem
-                name={chapter.name}
-                onMoveUp={index !== 0 ? () => { props.onMoveUp(index) } : null}
-                onMoveDown={index < arr.length - 1 ? () => { props.onMoveDown(index) } : null}
-                onDelete={() => { props.onDelete(index) }}
-                onClick={() => { props.onClick(index) }}
-                errors={props.errors ? props.errors[index] : {}}
-                />
-            </li>
-          )
+          props.chapters.map((chapter, index, arr) => {
+            const key = chapter.id || chapter.key
```

Comment:
Does this fallback work? When is this applied? Because `component.key`should be undefined, I suppose: https://github.com/facebook/react/issues/2429
Same goes for where ever else this is used.